CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Added
 
 - Put back secret pages-page
+- Texting feature under Messages admin tab created and hidden behind a feature flag
```

Comment:
No need to add it to the changelog if not visible :)

Reply:
Good call, removing now.